packages/api-derive/src/balances/fees.ts

```diff
@@ -26,21 +22,20 @@ import { drr } from '../util/drr';
  */
 export function fees (api: ApiInterface$Rx) {
   return (): Observable<DerivedFees> => {
-    return (api.queryMulti([
-      api.query.balances.creationFee,
-      api.query.balances.existentialDeposit,
-      api.query.balances.transactionBaseFee,
-      api.query.balances.transactionByteFee,
-      api.query.balances.transferFee
-    ]) as any as Observable<[BN, BN, BN, BN, BN]>).pipe(
-      map(([creationFee, existentialDeposit, transactionBaseFee, transactionByteFee, transferFee]) => ({
-        creationFee,
-        existentialDeposit,
-        transactionBaseFee,
-        transactionByteFee,
-        transferFee
-      } as DerivedFees)),
-      drr()
-    );
+    const {
+      creationFee,
+      existentialDeposit,
+      transactionBaseFee,
+      transactionByteFee,
+      transferFee
+    } = api.consts.balances;
```

Comment:
As per #1064, derives should support both api.consts.balances and api.query.balances. But maybe that's #1068's goal?

Reply:
Oh, yeah, it is actually. I was already working on the derive methods, just put it on hold. (...and got lost in test/compilation hell)